design-library/src/components/BccTabs/BccTabs.css

```diff
@@ -0,0 +1,62 @@
+@layer components {
+    .bcc-tabs {}
+
+    .bcc-tabs-bar {
+        @apply relative flex items-end gap-4 md:gap-8;
+    }
+
+    .bcc-tabs-bar::before {
+        @apply absolute bottom-0 inset-x-0 border-b-2 border-on-primary;
+        content: '';
+    }
+
+    @media (max-width: theme('screens.sm')) {        
```

Comment:
Not a big issue, but why wrap this in an `@media` instead of using the `sm:` prefix? It's what I've mostly done elsewhere, don't have a particular strong opinion on it although I do think less nesting is nicer.

Reply:
I am using media :) the prefix is just to not use a static value oh, and in this case it is max-width, vs @media being min-width